sunpy/data/data_manager/manager.py

```diff
@@ -172,10 +181,17 @@ def get(self, name):
         `KeyError`
             If ``name`` is not in the cache.
         """
-        return pathlib.Path(self._file_cache[name])
+        return pathlib.Path(self._file_cache[name][self._namespace])
 
     def _cache_has_file(self, urls):
         for url in urls:
             if self._cache._get_by_url(url):
                 return True
         return False
+
+    def _get_module(self, func):
+        # Made into a separate function for the ability to patch in tests
```

Comment:
I think this should get a docstring

Reply:
[Added](https://github.com/adwaitbhope/sunpy/blob/cefba371a5e761afe1a3b87ce32975590c4c9c94/sunpy/data/data_manager/manager.py#L192)